recipe/meta.yaml

```diff
@@ -30,6 +32,18 @@ requirements:
     #- pandas >=1.3
     # Optional lxml, pygraphviz, pydot, gdal packages provide additional functionality
 
+    # Steve:
+    # Do we want to add the following packages to this recipe?
```

Comment:
it is usually not a good idea to enforce optional dependencies, just add any optional dependencies together with `networkx` when packaging anything that needs them.

Reply:
Ok I've removed all the extras and pushed the changes.